src/fmeval/reporting/constants.py

```diff
@@ -180,13 +184,15 @@ class ListType(Enum):
     # General semantic robustness
     WER_SCORE: "Word error rate (WER) is a value between 0 and 1, and measures the difference between the model output on the unperturbed input and the output(s) on one or more perturbed versions of the same input. For more details on how word error rate is computed, see the [HuggingFace Article on Word Error Rate](https://huggingface.co/spaces/evaluate-metric/wer).",
     # Summarization semantic robustness
-    DELTA_ROUGE_SCORE: "The performance change of the ROUGE-N score is measured.",
-    DELTA_METEOR_SCORE: "The performance change of the METEOR score is measured.",
-    DELTA_BERT_SCORE: "The performance change of the BERTscore is measured.",
+    DELTA_ROUGE_SCORE: "Delta ROUGE-N score measures the change in Precision between the original and perturbed versions of the same input.",
```

Comment:
change in Rouge score. Same applies to METEOR and BERT Score, Exact Match score and F1 score.